avocado/utils/lv_utils.py

```diff
@@ -348,7 +348,7 @@ def lv_create(vg_name, lv_name, lv_size, force_flag=True):
     elif lv_check(vg_name, lv_name) and force_flag:
         lv_remove(vg_name, lv_name)
 
-    cmd = "lvcreate --size %s --name %s %s" % (lv_size, lv_name, vg_name)
+    cmd = "lvcreate --size %s --name %s %s -y" % (lv_size, lv_name, vg_name)
```

Comment:
I don't see this option in the `lvcreate -h` but it works well on Fedora as well as RHEL.7. Lets get this in and we can revert/improve it if some version does not support it.